game/src/score.rs

```diff
@@ -29,11 +29,11 @@ fn add_score(
     }
 }
 
-fn setup_score(
-    mut commands: Commands
-) {
-    commands.spawn((Score {
-        history: vec!(),
-    }, Name::new("Score")));
+fn setup_score(mut commands: Commands, mut score_q: Query<&mut Score>) {
+    if let Ok(mut score) = score_q.get_single_mut() {
+        score.history.clear();
```

Comment:
This clearing of the history isn't necessary, and obviously causes the history to only contain at most a single entry. That's probably not what `history` is meant to do. So it's probably better to remove this line.